### ISSUE TITLE
training problem of lstm

### ISSUE THREAD

**Issue by doobidoob** [AlexeyAB/darknet#3292]
Hi AlexeyAB.
Thanks for your support.

I tried to train "yolo_v3_tiny_pan_lstm" on some sequential dataset.
But, When I start training, "-nan" occurs from the beginning (except "No obj").
Like this...
![image](https://user-images.githubusercontent.com/41995698/58714626-10e7bd80-8400-11e9-9475-6f9d20a066b9.png)

I continued training, but the problem was not solved.
I used weight of yolov3-tiny.conv.14 and attached cfg file for training. 

[yolo_v3_tiny_pan_lstm_test.cfg.txt](https://github.com/AlexeyAB/darknet/files/3241764/yolo_v3_tiny_pan_lstm_test.cfg.txt)

Do you know why this problem happening?
Thanks in advance for your answer!

**Comment by AlexeyAB** [comment]
Try to use these params:
```
[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=16
subdivisions=4
width=512
height=512
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1


track=1
time_steps=3	# for 8GB GPU
augment_speed=3
sequential_subdivisions=4

learning_rate=0.001
burn_in=1000
max_batches = 10000

policy=sgdr
sgdr_cycle=10000
sgdr_mult=2
steps=4000,6000,8000,9000
#scales=1, 1, 0.1, 0.1
seq_scales=0.5, 1, 0.5, 1

```

**Comment by doobidoob** [comment]
@AlexeyAB 
Thanks for your reply.
I tried but, it's still not work....
![image](https://user-images.githubusercontent.com/41995698/58717314-ec8edf80-8405-11e9-94b2-3e8cf40d44cb.png)

Hmm....
I made training list in "xxx.data" sequentially like below.
Is this wrong?
![image](https://user-images.githubusercontent.com/41995698/58717660-b0a84a00-8406-11e9-90b2-5ccc778696bd.png)

I'm trying to solve this problem. but it's not easy.
Do you know where this problem occurs in the cfg or code...?
Thank you.

**Comment by AlexeyAB** [comment]
* What params do you use in the Makefile?
* What command do you use for training?
* Can you show output of commands?
```
nvcc --version
nvidia-smi
```
* What versions of CUDA, cuDNN and OpenCV do you use?
* Can you show content of files
obj.data
train.txt (first 20 lines)
bad.list
bad_label.list

* Did you use Yolo_mark for labeling? https://github.com/AlexeyAB/Yolo_mark

* Try to run training with flag ` -show_imgs` do you see the correct labels?

**Comment by doobidoob** [comment]
@AlexeyAB 
I solved it. Just my mistake.... It is a labeling problem in certain frame.
Sorry to bother you.
Thank you!
